code/game/gamemodes/clock_cult/clock_objects.dm

```diff
@@ -0,0 +1,1579 @@
+		user << "It has [stored_alloy]/[max_alloy] units of liquified replicant alloy stored."
+
+/obj/item/clockwork/clockwork_proselytizer/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/clockwork/component/replicant_alloy) && is_servant_of_ratvar(user) && uses_alloy)
+		if(stored_alloy >= max_alloy)
+			user << "<span class='warning'>[src]'s replicant alloy compartments are full!</span>"
+			return 0
+		modify_stored_alloy(10)
+		user << "<span class='brass'>You force [I] to liquify and pour it into [src]'s compartments. It now contains [stored_alloy]/[max_alloy] units of liquified alloy.</span>"
+		user.drop_item()
+		qdel(I)
+		return 1
+	..()
+
+/obj/item/clockwork/clockwork_proselytizer/afterattack(atom/target, mob/living/user, flag, params)
+	if(!target || !user)
+		return 0
+	if(user.a_intent == "harm" || !user.Adjacent(target))
+		return ..()
+	proselytize(target, user)
+
+/obj/item/clockwork/clockwork_proselytizer/proc/modify_stored_alloy(amount)
+	if(ratvar_awakens) //Summoning Ratvar doesn't make it free, but it might as well
+		amount = 1
+	stored_alloy = max(0, min(max_alloy, stored_alloy + amount))
+	return 1
+
+/obj/item/clockwork/clockwork_proselytizer/proc/proselytize(atom/target, mob/living/user)
+	if(!target || !user)
+		return 0
+	var/operation_time = 0 //In deciseconds, how long the proselytization will take
+	var/new_obj_type //The path of the new type of object to replace the old
+	var/alloy_cost = 0
+	var/valid_target = FALSE //If the proselytizer will actually function on the object
+	switch(target.type)
+		if(/turf/closed/wall, /turf/closed/wall/rust)
+			operation_time = 50
+			new_obj_type = /turf/closed/wall/clockwork
+			alloy_cost = 10
+			valid_target = TRUE //Need to change valid_target to 1 or TRUE in each check so that it doesn't return an invalid value
+		if(/turf/open/floor/plating, /turf/open/floor/plasteel)
+			operation_time = 30
+			new_obj_type = /turf/open/floor/clockwork
+			alloy_cost = 5
+			valid_target = TRUE
+	if(!uses_alloy)
+		alloy_cost = 0
+	if(!valid_target)
+		user << "<span class='warning'>[target] cannot be proselytized!</span>"
+		return 0
+	if(!new_obj_type)
+		user << "<span class='warning'>That object can't be changed into anything!</span>"
+		return 0
+	if(stored_alloy - alloy_cost < 0)
+		user << "<span class='warning'>You need [alloy_cost] replicant alloy to proselytize [target]!</span>"
+		return 0
+	user.visible_message("<span class='warning'>[user]'s [src] begins tearing apart [target]!</span>", "<span class='brass'>You begin proselytizing [target]...</span>")
+	playsound(target, 'sound/machines/click.ogg', 50, 1)
+	if(!do_after(user, operation_time, target = target))
+		return 0
+	if(stored_alloy - alloy_cost < 0) //Check again to prevent bypassing via spamclick
+		return 0
+	user.visible_message("<span class='warning'>[user]'s [src] disgorges a chunk of metal and shapes it over what's left of [target]!</span>", \
```

Comment:
This is producing:
clockwork nerd's **the** clockwork proselytizer disgorges a chunk of metal and shapes it over what's left of the floor!
you probably don't want the 'the' there